app-authz-uma-photoz/photoz-testsuite/src/test/java/org/keycloak/quickstart/uma/page/LogoutConfirmPage.java

```diff
@@ -0,0 +1,28 @@
+package org.keycloak.quickstart.uma.page;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static org.jboss.arquillian.graphene.Graphene.waitGui;
+
+/**
+ * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
+ */
+public class LogoutConfirmPage {
+
+    @FindBy(css = "input[type=\"submit\"]")
+    private WebElement confirmLogoutButton;
+
+    @FindBy(className = "instruction")
+    private WebElement infoMessage;
+
+    public void confirmLogout() {
+        waitGui().until().element(confirmLogoutButton).is().clickable();
+        confirmLogoutButton.click();
+    }
+
+    public boolean isUserLoggedOut() {
+        waitGui().until().element(infoMessage).is().present();
+        return "You are logged out".equals(infoMessage.getText());
+    }
```

Comment:
Just a nitpick. I don't think this is used anywhere.

Reply:
Yes, true. I removed this.

Reply:
Good catch BTV :)